### PR TITLE
Flanking multi starlane attack bonus (aka pincer attack) 

### PR DESCRIPTION
**Pull request by agrrr3**
[forum discussion](https://www.freeorion.org/forum/viewtopic.php?p=107253#p107253)

### CODE REVIEW

**Comment by agrrr3** [comment]
Ok feature works now as intended.

**Comment by agrrr3** [comment]
squashed and repushed. 

macosX build did not finish before, looks to me CI is broken(?)

File: default/scripting/policies/FLANKING.focs.txt [hunk 1]
```diff
@@ -7,31 +7,55 @@ Policy
...
+                          value = LocalCandidate.PreviousToFinalDestinationID
+                          condition = And [
+                              Fleet
+                              OwnedBy empire = RootCandidate.Owner
+                              (LocalCandidate.FinalDestinationID = RootCandidate.Fleet.FinalDestinationID)
+                              (LocalCandidate.ETA = 1)
+                              Contains condition = And [ Ship Armed ]
+                          ]
+                        ) > 1
+                       )
```
Comment (geoffthemedio) [inline default/scripting/policies/FLANKING.focs.txt]: de-indent by 4 spaces?

**Comment by agrrr3** [comment]
yikes. CI failures.. the ubuntu one looks like it could be that we are e.g. (wild guess) trying to log after starting termination and that is maybe already destroyed 
```
3: terminate called after throwing an instance of 'boost::wrapexcept<boost::log::v2_mt_posix::system_error>'
3:   what():  Failed to set TLS value: Invalid argument
3: 15:02:16.457379 {0x00007f7ff299e780} [debug] test : Process.cpp:261 : Process::Impl::Terminate done
3: 15:02:16.457429 {0x00007f7ff299e780} [warn] test : Process.cpp:263 : Process::Impl::Terminate got failure status 139
```

the snap build could not refresh lxd via snap..

**Comment by geoffthemedio** [comment]
> yikes. CI failures..

They all have green checkmarks for me...

**Comment by agrrr3** [comment]
> > yikes. CI failures..
> 
> They all have green checkmarks for me...

hm. now also for me, but 81bffba was definitely red. maybe they were automatically retried *head-scratch*?

**Pull request merged**